intro/integration/js.md

```diff
@@ -14,6 +18,8 @@ var skygear = require('skygear');
 import skygear from 'skygear';
 ```
 
+For most of the cases, this should be enough to get Skygear JS SDK running. If you are using React Native, Angular or Ionic with an old version, please follow the guides below.
```

Comment:
@b123400 Actually I think we can remove the old version instructions from guides to avoid cluttering it. Most people reading these sort of guides would be using latest version anyway.